libpromises/evalfunction.c

```diff
@@ -6990,6 +6990,10 @@ void ModuleProtocol(EvalContext *ctx, char *command, const char *line, int print
             StringSetAddSplit(tags, content, ',');
             StringSetAdd(tags, xstrdup("source=module"));
         }
+        else if (1 == sscanf(line + 1, "persistence=%ld", persistence))
```

Comment:
``scanf`` is the only one of the ``*f`` family of functions that we don't have a platform wrapper for.